prowler/providers/aws/services/opensearch/opensearch_service_domains_not_publicly_accessible/opensearch_service_domains_not_publicly_accessible.metadata.json

```diff
@@ -1,28 +1,27 @@
 {
   "Provider": "aws",
   "CheckID": "opensearch_service_domains_not_publicly_accessible",
-  "CheckTitle": "Check if Amazon Opensearch/Elasticsearch domains are set as Public or if it has open policy access",
+  "CheckTitle": "Check if Amazon Opensearch/Elasticsearch domains are publicly accessible",
   "CheckType": [
-    "Protect",
-    "Secure Access Management"
+    "Effects/Data Exposure"
   ],
   "ServiceName": "opensearch",
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "critical",
   "ResourceType": "AwsOpenSearchServiceDomain",
-  "Description": "Check if Amazon Opensearch/Elasticsearch domains are set as Public or if it has open policy access",
+  "Description": "Check if Amazon Opensearch/Elasticsearch domains are publicly accessible via their access policies.",
   "Risk": "Publicly accessible services could expose sensitive data to bad actors.",
   "RelatedUrl": "",
   "Remediation": {
     "Code": {
-      "CLI": "https://www.trendmicro.com/cloudoneconformity/knowledge-base/aws/Elasticsearch/elasticsearch-domain-exposed.html",
+      "CLI": "",
       "NativeIaC": "",
-      "Other": "https://docs.prowler.com/checks/aws/public-policies/public_3#fix---runtime",
+      "Other": "https://www.trendmicro.com/cloudoneconformity/knowledge-base/aws/Elasticsearch/elasticsearch-domain-exposed.html",
```

Comment:
This link returns a 404. Why are you removing the one from our knowledge base?

Reply:
Link was changed!